### DISCUSSION TITLE
Non-async git cloning

### DISCUSSION THREAD

**Discussion by ispringle** [wbthomason/packer.nvim#723] [Ideas]
I have very slow and very high latency (non-Starlink satellite) internet. One thing packer really struggles with for me are installs/updates/syncs. I've increased the git timeout to 600 seconds, but that doesn't always do the trick. I find things still fail for no good reason. I can -- and often do -- manually git clone into the proper pack/packer directory and almost never have an issue with the clone. I suspect the issue might be the large number of clones being done all at the same time. I wonder if it would be possible to either turn off async clones or to add a limit to the number of queued clones.

**Comment by wbthomason** [comment]
You can try setting `config.max_jobs` to a low number (even `1`) to limit the number of simultaneous clones. This has worked for others with slow connections in the past.

**Reply by dennis605** [reply]
I have an similar issue while I'm trying to install LunarVim. Can you please tell in which file I should put that "config.max_jobs" statement or the statement:
`local packer = require('packer')
packer.init {
    max_jobs = 1,
}
packer.startup(
    function(use)
        ...` 

I can really not find the file, where I should put it in. Thank you in advance

**Comment by snaeil** [comment]
I had the same problem within a bigger open wifi network.
Many pulls seemed to fail just randomly and I suspect the network not to allow too many parallel connections.

@wbthomason 's  answer fixed it for me:
 
```lua
local packer = require('packer')
packer.init {
    max_jobs = 1,
}
packer.startup(
    function(use)
        ...
```
